submission/distributor_test.go

```diff
@@ -43,7 +44,7 @@ func ExampleDistributor() {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	d, err := NewDistributor(sampleValidLogList(), buildStubCTPolicy(1), newLocalStubLogClient)
+	d, err := NewDistributor(sampleValidLogList(), buildStubCTPolicy(1), newLocalStubLogClient, monitoring.InertMetricFactory{})
```

Comment:
All these tests could just use `nil` now, but fine as-is.

Reply:
Ack